lib/Fmt_ast.ml

```diff
@@ -4540,7 +4540,7 @@ let fmt_toplevel c ctx itms =
   let break_struct = c.conf.break_struct || is_top ctx in
   let fmt_item c ~last = function
     | `Item i ->
-        maybe_disabled c i.pstr_loc []
+        maybe_disabled c i.pstr_loc (Ast.Structure_item.attributes i)
```

Comment:
This allows:
```ocaml
let x=unformatted
[@@ocamlformat "disable"]
```

But not:
```ocaml
type t = { x : int }
[@@ocamlformat "field-space=loose"]
```

The first case seems very useful to me, it'll definitely be used. Someone will ask for the second for consistency.

Reply:
The second case already works for me.

Reply:
You are right !